### PR TITLE
Docs/staking with exodus wallet

### PR DESCRIPTION
**Pull request by remjx**
#### Problem

Exodus wallet was not on the list of Supported Wallets for Staking

#### Summary of Changes

Added it with link to their FAQ.

### CODE REVIEW

File: docs/src/staking.md [hunk 1]
```diff
@@ -68,6 +68,8 @@ Check out our [guide to using SolFlare](wallet-guide/solflare.md) for details.
  Ledger Nano.
  [Staking commands using the Solana Command Line Tools](cli/delegate-stake.md).
 
+ - Exodus makes the process very simple. See their [FAQ](https://support.exodus.com/article/1551-solana-staking-faq) for details.
```
Comment (jstarry) [inline docs/src/staking.md:71]: I think we should add a note that it's not possible to select a validator to stake to with this wallet. Similar to how Binance and FTX work (which also aren't listed here)

Comment (remjx) [reply docs/src/staking.md:71]: Sure. I will update.

Comment (remjx) [reply docs/src/staking.md]: @jstarry updated,

Comment (jstarry) [reply docs/src/staking.md]: Thanks!

**Pull request merged**